homework_03/01_alexander_yordanov/main_task.py

```diff
@@ -0,0 +1,42 @@
+import math
+from enum import Enum
+
+class ReturnTypes(Enum):
+    NO_COMMON_POINTS = 1
+    TOUCHING = 2
+    INTERSECTING = 3
+    SAME = 4
+
+class Point: 
+    def __init__(self, x, y) -> None:
+        assert isinstance(x, float)
+        assert isinstance(y, float)
+
+        self.x = x
+        self.y = y
+
+class Circle:
+    def __init__(self, center, r) -> None:
```

Comment:
You could use radius instead of r but not a big deal.

Reply:
Actually i preffer "r" instead of "radius"